Source/WebKit/UIProcess/WebsiteData/Cocoa/WebsiteDataStoreCocoa.mm

```diff
@@ -735,7 +735,7 @@ static NavigatingToAppBoundDomain schemeOrDomainIsAppBound(const String& host, c
         if (hasInitializedManagedDomains && forceReinitialization != ForceReinitialization::Yes)
             return;
         static const auto maxManagedDomainCount = 10;
-        NSArray<NSString *> *crossSiteTrackingPreventionDisabledDomains;
+        NSArray<NSString *> *crossSiteTrackingPreventionDisabledDomains = nil;
```

Comment:
I won't quarrel with a nil initializer, but I will share that RetainPtr<NSArray<NSString *>> would make this style of bug impossible.